scenarios/Makefile

```diff
@@ -14,6 +23,14 @@ CURRENT_USER       := $(shell id -u):$(shell id -g)
 # Set to devel to use in-development versions of components
 UP4_ENV ?= stable
 
+# Build sdfabric-onos image as part of the deps step
+BUILD_SDFABRIC ?= true
+ifeq ($(UP4_ENV),devel)
+SDFABRIC_DOCKER_TAG := master
+else
+SDFABRIC_DOCKER_TAG := stable
+endif
```

Comment:
Since the image is built locally and never pushed, do we care about this distinction in the tag?

Reply:
I used this distinction because in the case we don't specify some of the `_VERSION`, we fallback on the ones specified in the sdfabric-onos repo.